### ISSUE TITLE
Doesn't work when not run from within the exploded zip/tar

### ISSUE THREAD

**Issue by dwijnand** [ajermakovics/jvm-mon#8]
```bash
$ opt/jvm-mon-0.1/bin/jvm-mon
objc[39393]: Class JavaLaunchHelper is implemented in both /Users/dnw/.jenv/versions/1.8/bin/java and /Library/Java/JavaVirtualMachines/jdk1.8.0_112.jdk/Contents/Home/jre/lib/libinstrument.dylib. One of the two will be used. Which one is undefined.

node.js:376
        callback();
        ^
module.js:341: Error: Cannot find module '/Users/dnw/opt/jvm-mon-0.1/bin/src/dist/jvm-mon.js'

$ cd opt/jvm-mon-0.1
$ bin/jvm-mon
objc[39418]: Class JavaLaunchHelper is implemented in both /Users/dnw/.jenv/versions/1.8/bin/java and /Library/Java/JavaVirtualMachines/jdk1.8.0_112.jdk/Contents/Home/jre/lib/libinstrument.dylib. One of the two will be used. Which one is undefined.~
$ # that worked
```

**Comment by ajermakovics** [comment]
I suspected this was the case. Ideally the bin/jvm-mon script would handle this by changing current working directory. Unfortunately that script is generated by Gradle on every build and tweaking it is tricky

**Comment by sergiotudela** [comment]
The same for here.

```
$ cd opt/jvm-mon-0.1
$ bin/jvm-mon
```

It worked for me.

**Comment by ajermakovics** [comment]
Added a fix in the new version (0.2) to allow running from other directories

**Comment by dwijnand** [comment]
Works great. Thanks @ajermakovics.
